stories/toast.stories.js

```diff
@@ -289,16 +287,16 @@ export const Usage = () => {
       <div className='mb-2'>
         <Button variant='outlined' size='lg' onClick={() => setPopUp(true)}>Click me</Button>
       </div>
-      { popUp &&
-        <Toast
-          type='semantic-light'
-          color='success'
-          title='Pop-up Success' 
-          button={buttonP}
-          icon={<CheckBadge size='lg'/>} 
-          onClose={() => setPopUp(false)}
-        />
-      }
+      <Toast
+        type='semantic-light'
+        color='success'
+        title='Pop-up Success' 
+        button={buttonP}
+        icon={<CheckBadge size='lg'/>} 
+        display={popUp}
```

Comment:
maybe can also rename `display` to `open` so it sounds more coherent with the close handler `onClose` lol